js/data.js

```diff
@@ -14,3 +14,21 @@ function getPokemon(){
 }
 
 document.addEventListener('DOMContentLoaded',getPokemon())
+
+function getPokemonByType(type){
+  const xhr=new XMLHttpRequest()
+  xhr.open('GET','https://pokeapi.co/api/v2/type/'+ type + '?limit=9999')
```

Comment:
FYI: I would have a conditional to check if `type` exists and/or a default value for `type`